src/tools/getFailureLogs.ts

```diff
@@ -86,79 +90,85 @@ export async function getFailureLogs(args: {
       ],
     };
   }
-
+  let response;
   // eslint-disable-next-line no-useless-catch
   try {
     for (const logType of validLogTypes) {
       switch (logType) {
         case AutomateLogType.NetworkLogs: {
-          const logs = await retrieveNetworkFailures(args.sessionId);
+          response = await retrieveNetworkFailures(args.sessionId);
           results.push({
             type: "text",
             text:
-              logs.length > 0
-                ? `Network Failures (${logs.length} found):\n${JSON.stringify(logs, null, 2)}`
-                : "No network failures found",
+              response.message ||
```

Comment:
directly return the result of these conditionals in the response (for other methods as well)